source/ui/netsender.c

```diff
@@ -165,6 +165,8 @@ static struct in_addr find3DS(int retries)
 				break;
 			}
 		}
+
+		svcSleepThread(1e6);
```

Comment:
This is actually a `double` floating point constant. I'd advise you to write out the zeroes `1000000`. Actually I'd make this 10ms and not 1ms: `10000000`